### PR TITLE
use vscodium instead of vscode

### PR DESCRIPTION
**Pull request by PalinuroSec**
TL;DR
same experience and same plugins. no default telemetry. true open source binaries. compiled from official code.
more info at https://vscodium.com

vscodium is a free and open source binary distribution of the vscode source code. microsoft provides its own binaries of vscodium, but even if vscodium itself is open source, the official binaries are not, and vscodium is a community effort to fill this gap and provide a true open binary distribution of vscode.

vscodium is 100% compatible with vscode, the plugins are taken from the official repo and the code is actually the same. all the vscodium project does is to provide some custom build scripts to automatically pick the latest vscode version, disable telemetry and proprietary branding and build all the binary files.

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
@PalinuroSec could you fix the certificate for repository.vscodium.com?

**Comment by PalinuroSec** [comment]
just fixed, sorry for the inconvenience

**Comment by williamstein** [comment]
No longer relevant, since we don't install vscode anymore.  Users can do so, of course.

**Comment by MohamedElashri** [comment]
For people coming to this issue from Google. You can install `codium` instead of `code` that was used before. 

Minimal example for building new image
```
FROM sagemathinc/cocalc

# Add vscodium repos
RUN curl https://repository.vscodium.com/pub.gpg | gpg --dearmor > /etc/apt/trusted.gpg.d/vscodium.gpg; \
    curl https://dbeaver.io/debs/dbeaver.gpg.key | gpg --dearmor > /etc/apt/trusted.gpg.d/dbeaver.gpg; \
    echo 'deb https://repository.vscodium.com/debs/ vscodium main' > /etc/apt/sources.list.d/vscodium.list; \
    echo "deb https://dbeaver.io/debs/dbeaver-ce /" > /etc/apt/sources.list.d/dbeaver.list
# Update everything
RUN apt-get update;apt-get -y install apt-transport-https; apt-get -y dist-upgrade ; rm -rf /var/lib/apt/lists/*

# Install additional software
RUN apt-get update; \
    apt-get -y install \
        codium nano geogebra
```
